From 20aff7e2ac45cf02de83929443aaa8407bccb89d Mon Sep 17 00:00:00 2001
From: Steve Hay <steve.m.hay@googlemail.com>
Date: Mon, 10 Aug 2015 13:12:07 +0100
Subject: Fix test count in t/base/rs.t

Commit da902b5900 cherry-picked 5fe499a8e2, but I got the conflict
resolution wrong. Now resolved correctly after looking at blead commit
0b81c0dda6, which made corrections to the test counting/skipping.

Bug-Debian: https://bugs.debian.org/822336
Patch-Name: fixes/5.20.3/test_count_base_rs.diff
---
 t/base/rs.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/base/rs.t b/t/base/rs.t
index 0035cc528..8e84ad041 100644
--- a/t/base/rs.t
+++ b/t/base/rs.t
@@ -1,7 +1,7 @@
 #!./perl
 # Test $/
 
-print "1..49\n";
+print "1..39\n";
 
 $test_count = 1;
 $teststring = "1\n12\n123\n1234\n1234\n12345\n\n123456\n1234567\n";
@@ -32,7 +32,6 @@ open TESTFILE, "<./foo";
 binmode TESTFILE;
 test_record(*TESTFILE);
 close TESTFILE;
-test_bad_setting();
 $test_count_end = $test_count;  # Needed to know how many tests to skip
 
 $/ = "\n";
